build/build.sh

```diff
@@ -61,12 +60,23 @@ if [ -n "$TRAVIS_TAG" ]; then
     # Example: v1.10.0-custom maps to 1.10.0-custom
     VERSION="${TRAVIS_TAG#v}"
 else
-    VERSION="dev"
+    ## Counting dots in CURRENT_BRANCH. If it is
+    ## release branch then it will have more than
+    ## one dot(.) because we are following release
+    ## branch naming convention as v1.10.x, v0.9.x
+    ## v2.0.x, v1.0.x
+
+    ## If we compare with master then local building
+    ## will have unknown version
+    dot_count=$(echo "${CURRENT_BRANCH}" | grep -o "\." | wc -l)
```

Comment:
Can you post what the version will look like for the following branches:
- v1.11.x
- v1.11.x-ee
- v1.11.0
- v1.11.0-RC1

Reply:
If we raise a PR against below branch names then corresponding VERSION values
**BRANCH_NAME**    ---     **VERSION**
```sh
v1.11.x                          ---     1.11.x-dev
v1.11.x-ee                     ---     1.11.x-ee-dev
v1.11.0                          ---     1.11.0-dev
v1.11.0-RC1                 ---     1.11.0-RC1-dev
```
Above **VERSION** will be generated only if TRAVIS_TAG is [empty](https://github.com/openebs/cstor-operators/blob/3b13054649044aae928c258ee7c160dea8254bb6/build/build.sh#L57).

`v1.11.0 and v1.11.0-RC1` are tags. So when TRAVIS_TAG is set then VERSION value will be described as below
**TRAVIS_TAG**    ---     **VERSION**
```sh
v1.11.0-RC1            ---     1.11.0-RC1
v1.11.0                    ---      1.11.0    
```